<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortApplicationClass</class>
 <widget class="QMainWindow" name="SortApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SortApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget{background-color:rgb(243, 244, 246);}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5,1">
    <item>
     <widget class="QWidget" name="Top_Widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextBrowser" name="Main_textBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:22pt; color:#000000;&quot;&gt;SortApplication&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#565656;&quot;&gt;Your file classification and custom renaming tool&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="Func_Wiget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>18</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QWidget" name="file_Widget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
background-color:rgb(255, 255, 255);
border-radius:20px;
}</string>
         </property>
         <widget class="QLabel" name="iconLabel_FileUpload">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>25</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
border-image: url(:/SortApplication/images/iconfont_FileUpload.png);

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="textLabel_UploadFile">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Upload Your File</string>
          </property>
         </widget>
         <widget class="QWidget" name="fileSelectorWidget" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>200</width>
            <height>140</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#fileSelectorWidget {
    border: 2px dashed rgb(0, 170, 255);
    border-radius: 30px;
    padding: 10px;
    background-color: rgb(250, 250, 250);
}
</string>
          </property>
          <widget class="QLabel" name="iconLabel_CloudRecord">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>20</y>
             <width>40</width>
             <height>40</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
border-image: url(:/SortApplication/images/iconfont_CloudRecord.png);
    background-color: rgb(250, 250, 250);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="textLabel_select1">
           <property name="geometry">
            <rect>
             <x>4</x>
             <y>60</y>
             <width>192</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	color:black;
    background-color: rgb(250, 250, 250);
}</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#232323;&quot;&gt;Click to select files&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="textLabel_select2">
           <property name="geometry">
            <rect>
             <x>4</x>
             <y>75</y>
             <width>192</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	color:black;
    background-color: rgb(250,250,250);
}</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#232323;&quot;&gt;you can select multiply&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="textLabel_Selected">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>220</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>The File you Selected</string>
          </property>
         </widget>
         <widget class="QScrollArea" name="scrollArea_SelectedFile">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>260</y>
            <width>200</width>
            <height>180</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>200</width>
             <height>180</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="function_Widget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
background-color:rgb(255, 255, 255);
border-radius:20px;
}</string>
         </property>
         <widget class="QLabel" name="iconLabel_BasicFunction">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>25</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
border-image: url(:/SortApplication/images/iconfont_BasicFunction.png);

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="textLabel_SelectFunction">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Select the Function</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="Func_groupBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>201</width>
            <height>41</height>
           </rect>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
          <widget class="QRadioButton" name="sortRadioButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>60</width>
             <height>30</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton{
	color:black;
	font-size:14px;
}
</string>
           </property>
           <property name="text">
            <string>Sort</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="renameRadioButton">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>10</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton{
	color:black;
	font-size:14px;
}</string>
           </property>
           <property name="text">
            <string>Rename</string>
           </property>
          </widget>
         </widget>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>100</y>
            <width>231</width>
            <height>341</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="sortPage">
           <widget class="QPushButton" name="startSort_pushButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>310</y>
              <width>100</width>
              <height>26</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(0, 78, 235);
color:white;
border-radius:5px;
text-align: left;
padding-left: 12px;
}</string>
            </property>
            <property name="text">
             <string>StartFunc</string>
            </property>
            <property name="icon">
             <iconset resource="SortApplication.qrc">
              <normaloff>:/SortApplication/images/iconfont_StartOperation.png</normaloff>:/SortApplication/images/iconfont_StartOperation.png</iconset>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="withdrawSort_pushButton">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>310</y>
              <width>100</width>
              <height>26</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(255, 85, 0);
color:white;
border-radius:5px;
text-align: left;
padding-left: 12px;
}</string>
            </property>
            <property name="text">
             <string>Withdraw</string>
            </property>
            <property name="icon">
             <iconset resource="SortApplication.qrc">
              <normaloff>:/SortApplication/images/iconfont_WithdrawOperation.png</normaloff>:/SortApplication/images/iconfont_WithdrawOperation.png</iconset>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QGroupBox" name="sort_groupBox">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>201</width>
              <height>101</height>
             </rect>
            </property>
            <property name="title">
             <string>GroupBox</string>
            </property>
            <widget class="QRadioButton" name="time_radioButton">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>10</y>
               <width>170</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton{color:black}</string>
             </property>
             <property name="text">
              <string>By the Time Points</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="type_radioButton">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>40</y>
               <width>170</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton{color:black}</string>
             </property>
             <property name="text">
              <string>By the File Types</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="size_radioButton">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>70</y>
               <width>170</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton{color:black}</string>
             </property>
             <property name="text">
              <string>By the File Size</string>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QWidget" name="renamePage">
           <widget class="QPushButton" name="startRename_pushButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>310</y>
              <width>100</width>
              <height>26</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(0, 78, 235);
color:white;
border-radius:5px;
text-align: left;
padding-left: 12px;
}</string>
            </property>
            <property name="text">
             <string>StartFunc</string>
            </property>
            <property name="icon">
             <iconset resource="SortApplication.qrc">
              <normaloff>:/SortApplication/images/iconfont_StartOperation.png</normaloff>:/SortApplication/images/iconfont_StartOperation.png</iconset>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="withdrawRename_pushButton">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>310</y>
              <width>100</width>
              <height>26</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color:rgb(255, 85, 0);
color:white;
border-radius:5px;
text-align: left;
padding-left: 12px;
}</string>
            </property>
            <property name="text">
             <string>Withdraw</string>
            </property>
            <property name="icon">
             <iconset resource="SortApplication.qrc">
              <normaloff>:/SortApplication/images/iconfont_WithdrawOperation.png</normaloff>:/SortApplication/images/iconfont_WithdrawOperation.png</iconset>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="history_Widget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
background-color:rgb(255, 255, 255);
border-radius:20px;
}</string>
         </property>
         <widget class="QLabel" name="iconLabel_OperationRecord">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>25</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
border-image: url(:/SortApplication/images/iconfont_OperationRecord.png);

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="textLabel_HistoryRecord">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>The History Record</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="Bottom_Widget" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>Main_textBrowser</tabstop>
  <tabstop>scrollArea_SelectedFile</tabstop>
  <tabstop>sortRadioButton</tabstop>
  <tabstop>renameRadioButton</tabstop>
 </tabstops>
 <resources>
  <include location="SortApplication.qrc"/>
 </resources>
 <connections/>
</ui>
